Zend/zend.c

```diff
@@ -831,6 +835,7 @@ void zend_startup(zend_utility_functions *utility_functions) /* {{{ */
 	zend_error_cb = utility_functions->error_function;
 	zend_printf = utility_functions->printf_function;
 	zend_write = utility_functions->write_function;
+	zend_write_escape = utility_functions->write_escape_function;
```

Comment:
Removed a cast here that was removed from the line above

Reply:
Nice!